marimbabot_audio/src/audio_from_lilypond.py

```diff
@@ -58,7 +71,7 @@ def create_audio_from_lilypond(sentence):
     return audio_filename
 
 # callback for vision node, includes audio publisher
-def callback_vision_results(data: String, audio_publisher):
+def callback_lilypond_sentence(data: String, audio_publisher):
```

Comment:
This isn't a callback anymore so consider renaming it.

Reply:
done